underfs/cosn/pom.xml

```diff
@@ -0,0 +1,77 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-underfs</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>alluxio-underfs-cosn</artifactId>
+  <name>Alluxio Under File System - Tencent Cloud cosn</name>
+  <description>Tencent Cloud cosn Under File System implementation</description>
+
+  <properties>
+    <build.path>${project.parent.parent.basedir}/build</build.path>
+    <hadoop.version>3.1.0</hadoop.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.qcloud.cos</groupId>
+      <artifactId>hadoop-cos</artifactId>
+      <version>3.1.0-5.8.5</version>
```

Comment:
You can remove the `version` tag here, and maven can use the `hadoop-cos` definition from root pom.xml dependencyManagement.